lambdas/ec2/reaper.py

```diff
@@ -9,6 +9,9 @@
 import boto3
 
 ec2 = boto3.resource('ec2')
+ec2_client = boto3.client('ec2')
```

Comment:
@greglarkin did you switch to using ec2_client here for consistency purposes? Or was there a technical reason as well?